.npmrc

```diff
@@ -0,0 +1 @@
+ignore-scripts=true
```

Comment:
I started getting an error about an opencollective postinstall script when I tried to install. Without ignoring scripts now, you can't install dependencies. This only started happening at the time I post this and I'm not sure why.